tests/strategies/external/utils.py

```diff
@@ -71,7 +71,7 @@ def assert_bid_offer_aggregator_commands_return_value(return_value, is_offer):
     assert return_value[command_name][
         "seller" if is_offer else "buyer"] == "forecast_pv"
     assert return_value[command_name][
-        "original_offer_price" if is_offer else "original_bid_price"] == 200.0
+        "original_price" if is_offer else "original_price"] == 200.0
```

Comment:
Minor, but I think that `return_value[command_name]["original_price"]` should suffice

Reply:
@spyros thanks for spotting this, I have overlooked it, done!